frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -0,0 +1,168 @@
+      const name = `${testName}-${_.kebabCase(kind)}`; // end name with '&' to force test failures
+
+      const createResource = () => {
+        cy.log(`Create ${kind}: ${name}`);
+        cy.visit(
+          `${namespaced ? `/k8s/ns/${testName}` : '/k8s/cluster'}/${resource}?name=${testName}`,
+        );
+        listPage.clickCreateYAMLButton();
+        if (resourcesWithCreationForm.has(kind)) {
+          cy.byTestID('yaml-link').click();
+        }
+        yamlEditor.isLoaded();
+        let newContent;
+        // get, update, and set yaml editor content.
+        yamlEditor.getEditorContent().then((content) => {
+          newContent = _.defaultsDeep(
+            {},
+            { metadata: { name, labels: { [testLabel]: testName } } },
+            safeLoad(content),
+          );
+          yamlEditor.setEditorContent(safeDump(newContent, { sortKeys: true }));
+          cy.log('creates a new resource instance');
+          yamlEditor.clickSaveCreateButton();
+          cy.get(errorMessage).should('not.exist');
+        });
+      };
+
+      const deleteResource = () => {
+        cy.log(`Delete ${name}`);
+        cy.visit(`${namespaced ? `/k8s/ns/${testName}` : '/k8s/cluster'}/${resource}`);
+        listPage.filter.byName(name);
+        listPage.rows.countShouldBe(1);
+        listPage.rows.clickKebabAction(name, deleteHumanizedKind(kind));
+        modal.shouldBeOpened();
+        modal.submit();
+        modal.shouldBeClosed();
+        listPage.rows.shouldNotExist(name);
+      };
+
+      before(() => {
+        createResource();
+      });
+
+      after(() => {
+        deleteResource();
+      });
+
+      it('displays detail view for newly created resource instance', () => {
+        cy.url().should('include', `/${name}`);
+        detailsPage.titleShouldBe(name);
+      });
+
+      it(`displays a list view for the resource`, () => {
+        cy.visit(
+          `${namespaced ? `/k8s/ns/${testName}` : '/k8s/cluster'}/${resource}?name=${testName}`,
+        );
+        if (namespaced) {
+          cy.log('has a working namespace dropdown on namespaced objects');
+          listPage.projectDropdownShouldExist();
+          listPage.projectDropdownShouldContain(testName);
+        } else {
+          cy.log('does not have a namespace dropdown on non-namespaced objects');
+          listPage.projectDropdownShouldNotExist();
+        }
+      });
+
+      it('search view displays created resource instance', () => {
+        cy.visit(
+          `/search/${
+            namespaced ? `ns/${testName}` : 'all-namespaces'
+          }?kind=${kind}&q=${testLabel}%3d${testName}&name=${name}`,
+        );
+        listPage.rows.shouldExist(name);
+
+        cy.log('link to to details page');
+        listPage.rows.clickRowByName(name);
+        cy.url().should('include', `/${name}`);
+        detailsPage.titleShouldBe(name);
+      });
+
+      it('edits the resource instance', () => {
+        if (kind !== 'ServiceAccount') {
```

Comment:
Prefer an early return. Ideally we should log something as well.

```suggestion
        if (kind === 'ServiceAccount') {
          return;
        }
```

Is there a reason we don't edit service accounts?

Reply:
fixed.  This was a copy-n-paste from protractor test, removed the test/early return for 'ServiceAccount` and it ran fine with the rest of the k8 crud tests in Cypress.